force-app/main/default/classes/NoteController.cls

```diff
@@ -1,39 +1,96 @@
-public with sharing class NoteController {
-    @AuraEnabled
-    public static Id saveData(String label, String description ) {
-        try {
-            
-            Note__c newNote = new Note__c();
-            newNote.Label__c = label;
-            newNote.Description__c = description;
-            newNote.Created_Date__c = Date.today();
+    public with sharing class NoteController {
+        
+        @AuraEnabled
+        public static Note__c saveData(String label, String description ) {
+            try {
+                System.debug(' Label=' + label + ', Description=' + description);
+                Note__c newNote = new Note__c();
+                newNote.Label__c = label;
+                newNote.Description__c = description;
+                newNote.Created_Date__c = Datetime.now();
+                
+                insert newNote;
+                System.debug('Record successfully created:' + newNote.Id);
+                return newNote;
+            }catch(Exception e) {
+                System.debug('Error saving data: ' + e.getMessage());
+                throw new AuraHandledException('Error saving data: An error occurred while saving the data.');
+            }
     
-            insert newNote;             
-            return newNote.Id;
-        } catch (Exception e) {
-            System.debug('Error saving data: ' + e.getMessage());
-            return null;
-            // return 'Error saving data: ' + e.getMessage();
-        }
     }
     
     @AuraEnabled
     public static String deleteNoteOnServer(String noteId) {
-        try {
-            List<Note__c> noteToDelete = [SELECT Id FROM Note__c WHERE Id = :noteId LIMIT 1];
-            if (!noteToDelete.isEmpty()) {
-                delete noteToDelete;
-                return 'Success';
-            } else {
-                return 'Note not found';
+        
+            try {
+                List<Note__c> noteToDelete = [SELECT Id FROM Note__c WHERE Id = :noteId LIMIT 1];
+                //List<Note__c> noteToDelete is collection and beacause  if does not use  List<Note__c>   It's  erorr Emppty()
+                if (!noteToDelete.isEmpty()) { 
+                    delete noteToDelete;
+                    return 'Success';
+                } else {
+                    return 'Note not found';
```

Comment:
you can make this method void and nothing to return then. In case of unsuccessful delete you can throw exception